DfE.FindInformationAcademiesTrusts/Pages/Trusts/Ofsted/CurrentRatings.cshtml

```diff
@@ -0,0 +1,72 @@
+@page "/trusts/ofsted/current-ratings"
+@using DfE.FindInformationAcademiesTrusts.Extensions
+@model CurrentRatingsModel
+
+@{
+  Layout = "Ofsted/_OfstedLayout";
+}
+
+<section class="govuk-!-margin-top-8 app-table-container" data-testid="ofsted-current-ratings-table">
+  <table class="govuk-table" data-module="moj-sortable-table">
+    <caption class="govuk-table__caption govuk-table__caption--m">Current ratings</caption>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header" aria-sort="ascending" data-testid="ofsted-current-ratings-school-name-header">
+          School name
+        </th>
+        <th scope="col" class="govuk-table__header govuk-body column-header-ofsted-rating" aria-sort="none" data-testid="ofsted-current-ratings-quality-of-education-header">
+          Quality of education
+        </th>
+        <th scope="col" class="govuk-table__header govuk-body column-header-ofsted-rating" aria-sort="none" data-testid="ofsted-current-ratings-behaviour-and-attitudes-header">
```

Comment:
aria-sort being "none" is that correct here?

Reply:
Yes, the sortable table modifies it when you click on the header.